kbase-extension/scss/partials/_appInput.scss

```diff
@@ -54,4 +54,41 @@
             }
         }
     }
+
+    // .kb-appInput__sequence
+    &__sequence {
+        // .kb-appInput__sequence__button_container
+        &__button_container {
+            padding: 0;
+            height: 100%;
+        }
+
+        // .kb-appInput__sequence__row
+        &__row {
+            width: 100%;
+            padding: 2px;
```

Comment:
I just copied these straight from the relevant files, didn't update to `em`/`rem` yet.